people/praveen-kumar.md

```diff
@@ -0,0 +1,11 @@
+Praveen Kumar
+------------------
+
+![](photos/praveen-kumar.png)
+
+* [Github](http://github.com/praveenk01)
+
```

Comment:
You can cleanup the lines here too, use an editor like Sublime Text and it will do this automagically for you!